build/gulp/generator/ts-configs/angular.tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "target": "ES2016",
+        "module": "ESNext",
```

Comment:
We can extract the base part as it did [here](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-scheduler/tsconfig.json#L2)